Microsoft.Packaging.Tools/tasks/targets/Microsoft.Packaging.Tools.Trimming.targets

```diff
@@ -22,7 +22,12 @@
       <TrimFilesRootFiles Include="hostpolicy.dll;libhostpolicy.dylib;libhostpolicy.so" />
 
       <!-- treat platform packages as trimmable -->
-      <TrimmablePackages Include="$(PackageConflictPreferredPackages)" />
+      <TrimmablePackages Condition="'$(DontTrimNETCoreAppRuntime)' != 'true'" Include="$(PackageConflictPreferredPackages)" />
```

Comment:
Neither Skip nor Enable made sense to me in this case.  I spent a good bit of time thinking about this one and couldn't come up with a name I liked better.  If you have a concrete suggestion that you think is better feel free to suggest that.